tests/Drupal/good/good.php

```diff
@@ -1124,7 +1124,7 @@ public function test8() {}
 
 }
 
-t('Some long mulit-line 
+t('Some long mulit-line
```

Comment:
My editor automatically removed trailing spaces. Was this space needed? If so, this change should be reverted.

Reply:
That's intersting, because all the tests passed without it.

Reply:
Reverted.